pandas-stubs/core/frame.pyi

```diff
@@ -13,8 +13,10 @@ from typing import (
     Iterator,
     Sequence,
     Generator,
+    Iterable,
+    Series,
 )
-from typing_extensions import Literal
+from typing_extensions import Literal, Scalar
```

Comment:
What is `Scalar`? I cannot find anything about it. If you wanted to have `Union[int, float]`, then just use `float`; it counts for both.

Reply:
I pulled that term directly from the documentation - ["value - scalar, dict, Series, or DataFrame
Value to use to fill holes (e.g. 0), alternately a dict/Series/DataFrame of values specifying which value to use for each index (for a Series) or column (for a DataFrame). Values not in the dict/Series/DataFrame will not be filled. This value cannot be a list."](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.fillna.html?highlight=dataframe%20fillna#pandas.DataFrame.fillna)

TBH, I'm not sure if just allowing a float would work.